app.js

```diff
@@ -25,38 +25,38 @@ headSelect.addEventListener('change', () => {
 });
 
 middleSelect.addEventListener('change', () => {
-    // > set the character middle from the value of the middle <select>
-    // > redisplay the character
+    character.middle = middleSelect.value;
+    displayCharacter();
 });
 
 pantsSelect.addEventListener('change', () => {
-    // > set the character head from the value of the head <select>
-    // > redisplay the character
+    character.pants = pantsSelect.value;
+    displayCharacter();
 });
 
 function displayCharacter() {
     headImage.src = 'assets/character/' + character.head + '-head.png';
-    // > set the middle and pants images .src property based on the character middle and pants properties
+    middleImage.src = 'assets/character/' + character.middle + '-middle.png';
+    pantsImage.src = 'assets/character/' + character.pants + '-pants.png';
 }
 
 function displayPhrases() {
-    // clear the list
     phraseList.innerHTML = '';
 
     for (const phrase of character.phrases) {
-        // > create an <li> element
-        // > set the text of the li to the phrase
-        // > append the <li> to the phrases list
+        const li = document.createElement('li');
+        li.textContent = phrase;
+        phraseList.append(li);
```

Comment:
creating an HTML element based on our state information then inserting it into the document